### PR TITLE
Create config loader

### PR DESCRIPTION
**Pull request by Jordancooke**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (kingmushroom) [inline serverless.yml]: Do you need the plugin in here?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM python:3.7-slim
+
+COPY dev-requirements.txt /
+
+RUN apt-get update -y &&\
+    apt-get install git -y
+RUN pip install -r /dev-requirements.txt
+RUN apt-get update -y &&\
+    apt-get install curl -y &&\
+    curl -sL https://deb.nodesource.com/setup_10.x | bash &&\
```
Comment (kingmushroom) [inline Dockerfile]: not sure if its this section, but do you need to also install serverless-latest-layer-version into the container?

File: serverless.yml (hunk 1 above)
Comment (Jordancooke) [reply serverless.yml]: Really good point, I will also need to set-up a pipeline in concourse! Good spot

**Comment by Jordancooke** [comment]
The changes Mike has suggested have been implemented, I have set-up the concourse pipeline and will require config-loader to be merged to full test it.

**Pull request merged**